examples/custom-server-express/pages/posts.js

```diff
@@ -1,19 +1,17 @@
-import React, { Component } from 'react'
+import React from 'react'
 
-export default class extends Component {
-  static getInitialProps({ query: { id } }) {
-    return { postId: id }
-  }
+export default function(props) {
+  return (
+    <div>
+      <h1>My {props.id} blog post</h1>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua.
+      </p>
+    </div>
+  )
+}
 
-  render() {
-    return (
-      <div>
-        <h1>My blog post #{this.props.postId}</h1>
-        <p>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-          eiusmod tempor incididunt ut labore et dolore magna aliqua.
-        </p>
-      </div>
-    )
-  }
+export async function getServerSideProps({ query: { id } }) {
```

Comment:
Should use `getStaticProps` without reading the querystring.

Reply:
fix

Reply:
Fixed by removing the page.